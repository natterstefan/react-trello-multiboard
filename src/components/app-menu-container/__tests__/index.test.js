import React from 'react'
import { shallow } from 'enzyme'

import AppMenuContainer from '../'

describe('Component/AppMenuContainer', () => {
  const Child = () => <span>Child Content</span>

  test('should render without throwing an error', () => {
    const wrapper = shallow(
      <AppMenuContainer>
        <Child />
      </AppMenuContainer>,
    )
    expect(wrapper.dive()).toMatchSnapshot()
    expect(wrapper.find(Child).length).toEqual(1)
  })
})
