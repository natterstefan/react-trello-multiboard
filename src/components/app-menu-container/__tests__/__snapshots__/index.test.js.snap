// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component/AppMenuContainer should render without throwing an error 1`] = `
<div
  className="AppMenuContainer-root-1"
>
  <div
    className="AppMenuContainer-appFrame-3"
  >
    <WithStyles(AppBar)
      className="AppMenuContainer-appBar-4"
    >
      <WithStyles(Toolbar)
        disableGutters={true}
      >
        <WithStyles(IconButton)
          aria-label="open drawer"
          className="AppMenuContainer-menuButton-7"
          color="inherit"
          onClick={[Function]}
        >
          <pure(Menu) />
        </WithStyles(IconButton)>
        <WithStyles(Typography)
          className="AppMenuContainer-flex-2"
          color="inherit"
          noWrap={true}
          variant="title"
        >
          Trello Multiboard
        </WithStyles(Typography)>
      </WithStyles(Toolbar)>
    </WithStyles(AppBar)>
    <WithStyles(Drawer)
      anchor="left"
      classes={
        Object {
          "paper": "AppMenuContainer-drawerPaper-9",
        }
      }
      open={false}
      variant="persistent"
    >
      <div
        className="AppMenuContainer-drawerHeader-10"
      >
        <WithStyles(IconButton)
          onClick={[Function]}
        >
          <pure(ChevronLeft) />
        </WithStyles(IconButton)>
      </div>
      <WithStyles(Divider) />
      <AppMenuContent />
    </WithStyles(Drawer)>
    <main
      className="AppMenuContainer-content-11 AppMenuContainer-content-left-12"
    >
      <div
        className="AppMenuContainer-drawerHeader-10"
      />
      <Child />
    </main>
  </div>
</div>
`;
